utils/prop-types/src/prop-types.js

```diff
@@ -0,0 +1,15 @@
+import pt from 'prop-types';
+const customPropTypes = {
+  /** Can render as a different type of component */
+  as: pt.oneOfType([pt.string, pt.elementType]),
+  children: pt.oneOfType([pt.arrayOf(pt.node), pt.node]),
+  header: pt.oneOfType([pt.arrayOf(pt.node), pt.node]),
+  icon: pt.shape({}),
+  list: pt.oneOfType([pt.arrayOf(pt.node), pt.node]),
+  main: pt.oneOfType([pt.arrayOf(pt.node), pt.node]),
+  nav: pt.oneOfType([pt.arrayOf(pt.node), pt.node]),
+  post: pt.oneOfType([pt.arrayOf(pt.node), pt.node]),
+  title: pt.oneOfType([pt.arrayOf(pt.node), pt.node]),
+};
+const extendedPropTypes = { ...pt, ...customPropTypes };
+export default extendedPropTypes;
```

Comment:
The name of our default export should always match the file name. Our file name here is `prop-types`

Reply:
**prop-types** is not a valid name. I leave it as propTypes instead.